editor/components/editor-global-keyboard-shortcuts/editor-mode.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
The `editor/components` folder is only for reusable editor components without "layout" information. I think the "mode" is a layout information because it only indicates which mode is "visible". Can we move this component to the `editor/edit-post` folder instead?